docs/setup_visualstudio.md

```diff
@@ -238,6 +238,13 @@ int main() {
 {: data-title="main.cpp" }
 
 
+### `getopt` library
```

Comment:
I moved and edited this to to be more in harmony with our recent changes.  Double check me. @zjweiss

Reply:
LGTM